packages/app/src/app/overmind/actions.ts

```diff
@@ -66,6 +66,7 @@ type ModalName =
   | 'deleteSandbox'
   | 'feedback'
   | 'forkServerModal'
+  | 'newSandbox'
```

Comment:
We have a new way to render the "New Sandbox" modal, which allows for props. I can show you with a later suggestion.

Reply:
removed :)